browser/ui/views/side_panel/playlist/playlist_side_panel_coordinator.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
@sangwoo108 fyi you've added the wrong license here somehow

Reply:
Oh! Thank you so much. I'll fix it